package.json

```diff
@@ -72,9 +72,9 @@
     "start": "yarn --cwd examples/official-storybook storybook",
     "test": "node ./scripts/test.js",
     "test:cli": "npm --prefix lib/cli run test",
-    "test:e2e": "cypress run",
     "test:e2e-framework": "ts-node --project=./scripts/tsconfig.json ./scripts/run-e2e.ts",
-    "test:e2e-gui": "concurrently --success first --kill-others \"cypress open\" \"yarn serve-storybooks\""
+    "test:integration": "cypress run",
```

Comment:
i personally prefer e2e for these, i think of "integration" as something in between unit & e2e. but i think these are full e2e tests.

Reply:
>  i think of "integration" as something in between unit & e2e

I 💯  agree with you, that why I pick up this name 😂 

I renamed it to have something different from e2e because with the latter we are bootstrapping new projects with CLIs and `sb init`. In what I called `integration` we are "simply" using the examples so it feels less end-to-end. Plus, NPM scripts suggest we are using all the examples but in fact, we only have a few tests based on `official-storybook`: https://github.com/storybookjs/storybook/tree/next/cypress/integration.

Do you think I should go with something like `test:e2e-examples`?

Reply:
Yea I think this counts as e2e, don't you? `test:e2e-examples` makes sense to me.

Reply:
Updated! 🚀